src/CouchDB.Driver/Types/CouchDocument.cs

```diff
@@ -43,20 +44,34 @@ public CouchDocument()
 
         [DataMember]
         [JsonProperty("_conflicts")]
-        public List<string> Conflicts { get; set; }
-    }
+        private readonly List<string> _conflicts;
 
-    internal static class CouchDocumentExtensions
-    {
-        public static void ProcessSaveResponse(this CouchDocument item, DocumentSaveResponse response)
+        [JsonIgnore]
+        public IReadOnlyCollection<string> Conflicts => _conflicts.AsReadOnly();
```

Comment:
Possible breaking change in type change. I like it, just marking for visibility

Reply:
Should I bump to 2.0.0 just for that? While reading the code I was thinking that conflicts are read-only.

Reply:
I'm not very strict about semver so I don't care